sample-compose/src/main/java/com/airbnb/mvrx/compose/sample/ComposeSampleActivity.kt

```diff
@@ -25,7 +27,32 @@ import com.airbnb.mvrx.compose.mavericksViewModel
 
 data class CounterState(
     val count: Int = 0,
-) : MavericksState
+) : MavericksState {
+    constructor(arguments: ArgumentsTest): this(count = arguments.count)
+}
+
+data class ArgumentsTest(val count: Int) : Parcelable {
```

Comment:
if you're able to add any actual unit tests that would be great, so we can catch regressions automatically in the future

Reply:
I can add unit tests, but we need whole setup for compose tests and that seems like a work for separate PR. Do you agree with that?

Reply:
I updated the test in sample-compose in this PR. I will work on more generic tests for mvrx-compose